templates/.releaserc.yaml.tpl

```diff
@@ -2,7 +2,7 @@
 preset: conventionalcommits
 branches:
 {{- if $prereleases }}
-  - name: {{ .Git.DefaultBranch }}
+  - name: {{ or (stencil.Arg "releaseOptions.prereleasesBranch") .Git.DefaultBranch }}
```

Comment:
`or` is going to cause this to just return true

I _think_ what you want is this:

```suggestion
  - name: {{ stencil.Arg "releaseOptions.prereleasesBranch" | default .Git.DefaultBranch }}
```

Reply:
Huh, I tested this against a repo and it seemed to pick things up correctly. Not sure if it's a recent change or what.

Reply:
Maybe `or` passes through the values instead of evaluating the condition then... that being said I'd _much_ prefer the default route as that's more standard.

Reply:
Totally fair. Updated. 

The lispy Go template functions had me in full lisp mode 😄